docs/workflows.md

```diff
@@ -101,63 +101,83 @@ Examples:
+Actions are blocks of code that are packed and ready to use in a workflow. There are various actions out of the box that can interact with the stored data (Run SQL query, Read Entity), external data (Server Request), deliver messages (Send Email), business logic (Execute JavaScript (Server)), or modules themselves (Execute JavaScript and Stop Execution, Refresh Grid) They only need parameters/configurations to work and deliver business logic. Usually, actions need some input parameters based on which they will execute their specific code. Some of the actions also have output tokens, the result of the code being executed, that can be used in the next actions. This will give you the ability to create complex flows and obtain your custom business logic. When the workflow ends you can attach the output tokens from the actions to the output parameters of the workflow.
 
-#### `How to action to a workflow`
+#### `How to Add an Action to a Workflow`
 
-Action can be added to from the top bar in the workflow interface from the Add action Button or Imported from another exported action by copy/pasting the JSON configuration.
+An Action can be added from the top bar in the workflow interface using the `Add Action` button or the [Import](#importing-an-action-configuration) button.
 
-Alternatively you can add a new action from the small icon set next to an existing action (or start block) when it is selected. The icon for the new action is the rectangle.
+Alternatively, you can add a new action from the small icon set next to an existing action (or start block) when it is selected. The icon for the new action is the rectangle.
 
-For both options an popup will open and the list of the available action will show. You can search on the top for a specific action while the list can get pretty big in time.  After you choose an action by clicking on it the configuration popup will show. Click on save at the bottom of the popup to save the action after you configure it. Configuring the action is not required at this point and it can be saved blank.
+For both options, a popup will open and the list of the available actions will show. You can search on the top for a specific action or browse the list by expanding the categories. After you choose an action by clicking on it, the configuration popup will show. Click on `Save` at the bottom of the popup to save the action after you configure it. Configuring the action is not required at this point and it can be saved blank.
```

Comment:
```suggestion
For both options, a popup will open and show the list of available actions. You can search on the top for a specific action or browse the list by expanding the categories. After you choose an action by clicking on it, the configuration popup will show. Click on `Save` at the bottom of the popup to save the action after you configure it. Configuring the action is not required at this point and it can be saved blank.
```